sdk/spring/spring-cloud-azure-feature-management/src/main/java/com/azure/spring/cloud/feature/management/targeting/ContextualTargetingContextAccessor.java

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.spring.cloud.feature.management.targeting;
+
+/**
+ * Interface for providing a Targeting Context to the `Microsoft.TargetingFilter`.
+ */
+public interface ContextualTargetingContextAccessor {
```

Comment:
nit: but would it be better if we call these two accessor classes, xxxConfigurer?

Reply:
The original class is already GA, so we will need to match that.